exporter/graph/graph_crawler.py

```diff
@@ -65,7 +71,20 @@ def _crawl(self, process: MetadataResource) -> ExperimentGraph:
         processes_to_crawl = GraphCrawler.flatten([self.metadata_service.get_derived_by_processes(i) for i in process_info.inputs])
 
         return reduce(lambda g1, g2: g1.extend(g2),
-                      map(lambda proc: self._crawl(proc), processes_to_crawl),
+                      map(lambda proc: self._crawl_up(proc), processes_to_crawl),
+                      partial_graph)
+
+    def _crawl_down(self, process: MetadataResource) -> ExperimentGraph:
```

Comment:
`_crawl_up` and `_crawl_down` look very similar apart from the input to `GraphCrawler.flatten`. Have you considered extracting this into a common function?